Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN apt-get -y update && \
 
 # We need version 16+ of NodeJS for `make package` to succeed.
 RUN apt-get -y update && \
-    apt-get -y dist-upgrade && \
+    apt-get -y upgrade && \
```

Comment:
The same command is executed twice here

Reply:
I'm going to refactor the Dockerfile in another PR. I just did the minimum here.

Reply:
Can you remove the duplicates anyway?

Reply:
I removed the duplicate lines